packages/components/tooltip/src/tooltip.tsx

```diff
@@ -17,6 +17,7 @@ import type {
 import { motion, AnimatePresence } from "@yamada-ui/motion"
 import type { PortalProps } from "@yamada-ui/portal"
 import { Portal } from "@yamada-ui/portal"
+import { useOutsideClick } from "@yamada-ui/react"
```

Comment:
please import from use-outside-click
```suggestion
import { useOutsideClick } from "@yamada-ui/use-outside-click"
```

Reply:
https://github.com/yamada-ui/yamada-ui/pull/1386/commits/309c3e9c238a4ab668fb3346b9a5126369fbbd31